src/main/java/backend/Season.java

```diff
@@ -56,15 +66,27 @@ public static Season fromJson(String jsonString) {
      * Currently, it fails fast, instead of catching errors. But this can be changed
      * depending on the project requirements.
      *
-     * @param filename A file-location of a json file containing a Season
+     * @param inputFile A scanner of a json file containing a Season
      * @return A season
      * @throws IOException throws if the file does not exist
      */
-    public static Season fromJsonFile(String filename) throws IOException {
-        Scanner sc = new Scanner(new File(filename));
-        String fileString = sc.nextLine();
+    public static Season readFromJsonFile(Scanner inputFile) throws IOException {
+        String fileString = inputFile.nextLine();
         return fromJson(fileString);
+    }
 
+    /**
+     * Writes json to a file, fails fast.
+     *
+     * @param filename File to write to
+     * @throws IOException throws in rare cases (I'm not sure when)
+     */
+    public void writeToJsonFile(String filename) throws IOException {
+        FileOutputStream out = new FileOutputStream(filename);
```

Comment:
See trello. Also, because of "json-database", the whole file needs to be rewritten every time anyway.